openshift/release/cron-nightly-ci-run.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: pipelines-triggers-nightly-ci-run
+spec:
+  failedJobsHistoryLimit: 1
+  successfulJobsHistoryLimit: 1
+  concurrencyPolicy: Replace
+  schedule: "0 0 * * *"
+  startingDeadlineSeconds: 200
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: nightly-ci-sa
```

Comment:
where are we creating this SA :thinking:

Reply:
we have already created the sa on the qe cluster